pkg/support/dump_others.go

```diff
@@ -28,10 +28,12 @@ import (
 
 func (d *agentDumper) DumpLog(basedir string) error {
 	logDir := logdir.GetLogDir()
-	if err := fileCopy(d.fs, path.Join(basedir, "logs", "agent"), logDir, "antrea-agent"); err != nil {
+	timeFilter := timestampFilter(d.since)
+
+	if err := directoryCopy(d.fs, path.Join(basedir, "logs", "agent"), logDir, "antrea-agent", timeFilter); err != nil {
 		return err
 	}
-	return fileCopy(d.fs, path.Join(basedir, "logs", "ovs"), logDir, "ovs")
+	return directoryCopy(d.fs, path.Join(basedir, "logs", "ovs"), logDir, "ovs", timeFilter)
```

Comment:
I just thought of ovs log has different time format:
```
2020-11-24T10:00:56.837Z|00036|memory|INFO|415440 kB peak resident set size after 10.0 seconds
2020-11-24T10:00:56.837Z|00037|memory|INFO|handlers:35 ofconns:1 ports:2 revalidators:13 rules:37 udpif keys:1
2020-11-24T10:01:00.680Z|00038|connmgr|INFO|br-int<->unix#0: 34 flow_mods 10 s ago (33 adds, 1 deletes)
```
So the timeFilter won't work

Reply:
My mistake. The ovs log file part is not well tested.

Reply:
I have checked the code again. This was by design, alougth i'm not sure it's the best solution. The timefilter is based on the timestamp records in the log filename, which was generated by klog. If no timestamp found in the filename, the whole log file is returned, and it applied to the ovs logs. I assumed it's much smaller than the agent/controller log and does not need to do timestamp filter.

Reply:
A bad assumption i guess.